bansa/src/main/kotlin/com/brianegan/bansa/Store.kt

```diff
@@ -5,8 +5,8 @@ import rx.Subscriber
 import rx.Subscription
 
 interface Store<S, A> {
-    val state: Observable<S>
-    fun getState(): S
+    val stateChanges: Observable<S>
+    val state: S
```

Comment:
Neat Kotlin trick :) I like how you're keeping the interface read-only while using a `var` under the hood.